src/components/interactive-map.js

```diff
@@ -41,41 +31,86 @@ const propTypes = Object.assign({}, StaticMap.propTypes, {
    */
   onChangeViewport: PropTypes.func,
 
-  /** Enables perspective control event handling */
-  perspectiveEnabled: PropTypes.bool,
```

Comment:
This will be a pretty big backwards compatibility breaker. Should we consider having a section of deprecated props at the bottom. At least check and warn?